packages/framework/src/Component/LocalCache/Exception/ValueCacheKeyNotFoundException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\LocalCache\Exception;
+
+class ValueCacheKeyNotFoundException extends LocalCacheException
+{
+    /**
+     * @param string $namespace
+     * @param string $valueKey
+     */
+    public function __construct(string $namespace, string $valueKey)
+    {
+        parent::__construct(sprintf('Value with keys "%s" and "%s" not found', $namespace, $valueKey));
```

Comment:
I would be more precise here
```suggestion
        parent::__construct(sprintf('Value with namespace "%s" and key "%s" not found', $namespace, $valueKey));
```

Reply:
ok